frontend/src/theme/primitives/colors.ts

```diff
@@ -0,0 +1,127 @@
+export const colors = {
+  transparent: "transparent",
+  current: "currentColor",
+  black: "#000000",
+  white: "#FFFFFF",
+
+  whiteAlpha10: "rgba(255, 255, 255, 0.04)",
+  whiteAlpha20: "rgba(255, 255, 255, 0.06)",
+  whiteAlpha30: "rgba(255, 255, 255, 0.08)",
+  whiteAlpha40: "rgba(255, 255, 255, 0.16)",
+  whiteAlpha50: "rgba(255, 255, 255, 0.24)",
+  whiteAlpha60: "rgba(255, 255, 255, 0.36)",
+  whiteAlpha70: "rgba(255, 255, 255, 0.48)",
+  whiteAlpha80: "rgba(255, 255, 255, 0.64)",
+  whiteAlpha90: "rgba(255, 255, 255, 0.80)",
+  whiteAlpha100: "rgba(255, 255, 255, 0.92)",
+
+  blackAlpha10: "rgba(0, 0, 0, 0.04)",
+  blackAlpha20: "rgba(0, 0, 0, 0.06)",
+  blackAlpha30: "rgba(0, 0, 0, 0.08)",
+  blackAlpha40: "rgba(0, 0, 0, 0.16)",
+  blackAlpha50: "rgba(0, 0, 0, 0.24)",
+  blackAlpha60: "rgba(0, 0, 0, 0.36)",
+  blackAlpha70: "rgba(0, 0, 0, 0.48)",
+  blackAlpha80: "rgba(0, 0, 0, 0.64)",
+  blackAlpha90: "rgba(0, 0, 0, 0.80)",
+  blackAlpha100: "rgba(0, 0, 0, 0.92)",
```

Comment:
Why rgba here when elsewhere we use transparentize?

Reply:
and where do these come from?

Reply:
Again...kindly stolen, but possibly should be removed.